cirq-core/cirq/ops/measurement_gate.py

```diff
@@ -153,7 +153,7 @@ def with_bits_flipped(self, *bit_positions: int) -> 'MeasurementGate':
             self.num_qubits(), key=self.key, invert_mask=tuple(new_mask), qid_shape=self._qid_shape
         )
 
-    def full_invert_mask(self):
+    def full_invert_mask(self) -> Sequence[bool]:
```

Comment:
nit: Change it to `Tuple[bool, ...]` instead to return a more specific type, which is also consistent with other methods like `invert_mask()` above.

```suggestion
    def full_invert_mask(self) -> Tuple[bool, ...]:
```